### DISCUSSION TITLE
How can we check the OpenMage version?

### DISCUSSION THREAD

**Discussion by kevalkavathiya** [OpenMage/magento-lts#3439] [Questions & Answers]
After updating Magento ver. 1.9.4.2 to openmage v19.5.0-rc5, How can I check the version and confirm it is up-to-date??

I updated using git rebase and it is showing completed.
![image](https://github.com/OpenMage/magento-lts/assets/41768026/ceb92d4f-6138-4f77-b611-85430b306d0d)

but on the admin side still, it is showing Magento ver. 1.9.4.2. is this right???
![image](https://github.com/OpenMage/magento-lts/assets/41768026/982b6266-ec8e-4154-9b19-7be67ae870d7)

**Comment by elidrissidev** [comment]
Something must've went wrong when rebasing, you are still on `1.9.4.2`.

Do you really need to rebase? Why not just do a regular merge everytime you want to update to newer versions?

Also on a side note, `v19.5.0` was already released last week, no need to keep using the rc version.

**Comment by fballiano** [comment]
I think cache wasnt flushed, or opcache it’s not refreshing the php scripts

**Comment by kevalkavathiya** [comment]
@elidrissidev and @fballiano thanks for the response but I change the update way and its works for me :)
